docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -227,7 +229,8 @@ Within `fhir-server-config.json`, the value of the `fhirServer/persistence/facto
 
 ### 3.3.1 The JDBC persistence layer
 When the FHIRPersistenceJDBCFactory is in use, the `fhirServer/persistence/datasources` property must specify a mapping from datastore-id values to Liberty datasource definitions. For example, here is the configuration for a datastore with id `default` that is configured for the `jdbc/fhir_default_default` datasource of type `postgresql`:
-```
+
+``` json
```

Comment:
the `json` hint seems to cause the github editor some trouble, since technically the snippet isn't valid json